test/models/project_media_5_test.rb

```diff
@@ -84,9 +84,6 @@ def setup
   end
 
   test "should have a media not not necessarily a project" do
```

Comment:
@melsawy, I think you should probably remove this whole test ("should have a media not not necessarily a project")

Reply:
Removed this test